### ISSUE TITLE
Multiple meningiomata

### ISSUE THREAD

**Issue by DLewisSmith** [obophenotype/human-phenotype-ontology#6758]
**Preferred term label:**
Multiple meningiomas

**Synonyms**
Multiple meningiomata
Multifocal meningiomata

**Definition (free text, please give PubMed ID)**
The occurrence of multiple distinct meningiomas in the same individual.

**Parent term (use hpo.jax.org/app)**
Meningioma HP:0002858

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM 101000
NEUROFIBROMATOSIS, TYPE II; NF2

**Your nano-attribution (ORCID)**
0000-0002-1735-8178

**Comment by pnrobinson** [comment]
added as HP:0033714
